docs/general/dns-providers.md

```diff
@@ -590,7 +590,7 @@ More strictly filtering policies with blocking — ads, marketing, tracking, cli
 |DNS-over-HTTPS|`https://ric.openbld.net/dns-query`|[Add to AdGuard](sdns://AgAAAAAAAAAAAAAPcmljLm9wZW5ibGQubmV0Ci9kbnMtcXVlcnk)|
 |DNS-over-TLS|`tls://ric.openbld.net`|[Add to AdGuard](sdns://AwAAAAAAAAAAAAAPcmljLm9wZW5ibGQubmV0)|
 
-#### dns0.eu
+### dns0.eu
```

Comment:
you need to place it above so that DNS providers are listed in an alphabetical order